#I2C device
#In case of UM232H-B use "UM232H-B"
#In case of Devantech USB-ISS use according serial device path (e.g. "/dev/ttyACM0")
i2c_device: "UM232H-B"
#i2c_device: "/dev/ttyACM0"

#Vectors of I2C addresses and corresponding names
pad_i2c_ids: [0x04,0x05,0x06,0x07]
pad_names: ["left-front","left-back","right-front","right-back"]
